modules/howtos/examples/collection-manager.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+use \Couchbase\ClusterOptions;
+use \Couchbase\Cluster;
+use \Couchbase\User;
+
+function getCollections($username, $password) {
+    print "create-collection-manager\n";
+
+	// tag::create-collection-manager[]
+    $options = new \Couchbase\ClusterOptions();
```

Comment:
Do we need to re-use the fully qualified name here if we've actually imported above with `use`?
I'm never sure about this with PHP 😕

Reply:
I think can omit, but may be clearer to keep it, and remove the explicit import (which I copied from a previous example...)